app-modules/integration-open-ai/src/Services/OpenAiGptTestService.php

```diff
@@ -45,13 +45,23 @@ public function __construct()
         $this->client = new OpenAI\Testing\ClientFake();
     }
 
+    public function getApiKey(): string
+    {
+        return 'test';
+    }
+
+    public function getApiVersion(): string
+    {
+        return '1.0.0';
+    }
+
     public function getModel(): string
     {
         return 'test';
     }
 
-    public function getDeployment(): ?string
+    public function getDeployment(): string
     {
-        return null;
+        return 'test';
```

Comment:
This is a fake deployment, so it should return null since none of the data is persisted. If it's null, we don't need to transfer the data to a new service when the assistant model gets updated.

Reply:
This doesn't actually interact with the API, no? It just seemed a bit strange to enforce a that this potentially returns null when all practical use cases should not.